content/v1/csidriver/features/powerscale.md

```diff
@@ -440,7 +449,7 @@ The user can also set the volume limit for all the nodes in the cluster by speci
 
 ## Node selector in helm template
 
-Now user can define in which worker node, the CSI node pod daemonset can run (just like any other pod in Kubernetes world.)For more information, refer to https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector 
+Now user can define in which worker node, the CSI node pod daemonset can run (just like any other pod in Kubernetes world.)For more information, refer to https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
```

Comment:
Change "user" to "users". Remove comma after node. Place period outside of parentheses. Insert space before "For"